src/main/java/com/royalrangers/controller/achievement/PlatoonAchievementController.java

```diff
@@ -21,99 +23,194 @@
     @Autowired
     private UserService userService;
 
-    @GetMapping("/approved/medal")
-    public ResponseResult getMedals() {
-        return ResponseBuilder.success(platoonAchievementService.getMedalRewards());
+    @GetMapping("/approved/star")
+    public ResponseResult getApprovedStarRewards() {
+        try {
+            return ResponseBuilder.success(platoonAchievementService.getApprovedStarRewards());
+        } catch (Exception ex) {
+            log.debug("Failed get all approved star rewards for current platoon", ex);
```

Comment:
Please change log level to error and as one of the parameters add exception object